armi/nuclearDataIO/tests/test_xsLibraries.py

```diff
@@ -59,122 +54,6 @@
 UFG_FLUX_EDIT = os.path.join(FIXTURE_DIR, "mc2v3-AA.flux_ufg")
 
 
-def copyInputForPmatrxAndGamsio(inpPath):
-    with open(inpPath, "r") as inp, open(
-        inpPath.replace(".inp", ".pmatrx.inp"), "w"
-    ) as pmatrxInp, open(inpPath.replace(".inp", ".gamiso.inp"), "w") as gamisoInp:
-        for line in inp:
-            pmatrxInp.write(line.replace("isotxs", "pmatrx"))
-            gamisoInp.write(line.replace(".isotxs", ".gamiso"))
-
-
-def createTestXSLibraryFiles(cachePath):
```

Comment:
Lazy question: The files mentioned in this test are still relevant to other tests correct?

Reply:
That's a good question.  I'm like 95% sure, yes.

Reply:
This isn't a unit test, but a script that creates MC2-3 inputs to generate XS libraries to be used in unit tests. It probably doesn't belong in a unit test file, but it seems like we might still want to retain the functionality in case we want to update these files in the future? e.g., if we use a new version of MC2-3 that changes the format of the library files?

Reply:
I really appreciate the context. Looks like we moved too fast. 

https://github.com/terrapower/armi/issues/1174